src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -91,8 +91,9 @@ export class CheckboxSelectColumn<T extends Slick.SlickData> implements Slick.Pl
 	private checkboxSelectionFormatter(row: number, cell: number, value: any, columnDef: Slick.Column<T>, dataContext: T): string {
 		const state = this.getCheckboxPropertyValue(row);
 		const checked = state.checked ? 'checked' : '';
+		const ariaLabelStatus = state.checked ? 'checked' : 'unchecked';
 		const enable = state.enabled ? '' : 'disabled';
-		return `<input type="checkbox" style="pointer-events: none;" tabIndex="-1" ${checked} ${enable}/>`;
+		return `<input type="checkbox" style="pointer-events: none;" tabIndex="-1" aria-label="checkbox ${ariaLabelStatus}" ${checked} ${enable}/>`;
```

Comment:
it doesn't seem right to use aria-label for control type and state. those should be announced regardless, I don't have my Mac with me today, how does voiceover announce the normal checkboxes?

Reply:
This is the original accessibility issue we had a few years back said the screenreader should announce the control, checkbox, followed by the status https://github.com/microsoft/azuredatastudio/issues/6726.

Here's how it announces a normal checkbox - it's "status, checkbox", so different order than what I have in this PR: 
![image](https://user-images.githubusercontent.com/31145923/225995383-bb1a1e30-2194-40a0-99b3-05888fe4bedc.png)